tests/test_base_models.py

```diff
@@ -1,6 +1,9 @@
 # Copyright © 2021 United States Government as represented by the Administrator of the National Aeronautics and Space Administration.  All Rights Reserved.
 
 import unittest
+
+from attr import Attribute
```

Comment:
This adds a new dependency to prog_models. Is there a way to do this without adding a new dependency. Does the python built in setattr() function serve the same purpose?

Reply:
I think this was erroneously added too while I was testing - removed.